doc/conf.py

```diff
@@ -81,9 +81,9 @@ def setup(app):
 master_doc = "index"
```

Comment:
please do not change unrelated part.

Reply:
black formatting issue , it changed few unrelated files , will remove it soon.